README.md

```diff
@@ -14,7 +14,7 @@
 
 A web platform for collaboration on MBSE and [Capella](https://www.eclipse.org/capella/) projects.
 
-**Copyright 2021 - 2023 [DB Netz AG](https://fahrweg.dbnetze.com/),
+**Copyright 2021 - 2023 [DB InfraGO AG](https://fahrweg.dbnetze.com/),
```

Comment:
```suggestion
**Copyright 2021 - 2023 [DB InfraGO AG](https://fahrweg.dbinfrago.com/fahrweg-de),
```

Reply:
Ooh good catch, though should I also update the copyright to 2024?

Reply:
Also good catch from you 😄 Feel free to also update the copyright year

Reply:
Not sure why I even asked, done 😆 ✅